Bottom: a1a9cfa9f0b3da7e4c7b4eac32d592f3195c8afa
Top:    e038ac18d56a25359cc0f01bdbd771a64942b07a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-15 19:42:27 +0000

hid/gtk: Add a guard against GC's being passed from another HID

Various other HIDs have this check.


---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index 51a6f22..45dc710 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -47,7 +47,6 @@ Vy2 (int y)
 
 typedef struct hid_gc_struct
 {
-  HID *me_pointer;
   GdkGC *gc;
 
   gchar *colorname;
@@ -74,7 +73,6 @@ ghid_make_gc (void)
   hidGC rv;
 
   rv = g_new0 (hid_gc_struct, 1);
-  rv->me_pointer = &ghid_hid;
   rv->colorname = Settings.BackgroundColor;
   return rv;
 }
